### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]:Name property is empty for document in "Push Image dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1410]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Push Image.
4. In "push Image" dialog ,tab to document and observe the name property using inspect.

**Actual Result:**
Name property is empty for the document..
Narrator is not announcing the value.value information for the document.

**Narrator behaviour:**
Tab

**NVDA behaviour:**
edit  read only
blank

**Expected Result:**
Name property should be simple and meaningful.
Name property for the document should be "Push Image".
Narrator should  announce the value.value for the document .

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)

![push image-document](https://user-images.githubusercontent.com/22956130/35685829-c858e4be-0790-11e8-8435-31ac8755d6cd.jpg)
![push image-document-nvda](https://user-images.githubusercontent.com/22956130/35685833-cb480416-0790-11e8-8e83-b0492083e611.jpg)

**Comment by Eskibear** [comment]
Determined by Eclipse IDE, out of the toolkit's scope

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=533976

**Comment by jdneo** [comment]
* **Importance of experience**: minimal. Very low usage based on telemetry
* **Quality of experience**: not impacted. Users can still see the property in the `Value` field
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.


#### Workaround
Users can still see the property in the `Value` field
![image](https://user-images.githubusercontent.com/6193897/44139930-2cfb5f06-a0ab-11e8-87d4-c61ea51b377e.png)

**Comment by WiproPSAccTest** [comment]
Agreed with the resolution . Can you please remove A11YExclusion, wontfix, Resolved, notreproduce labels wherever applicable and add ExclusionAgreed

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749877
